src/typeDefs/gqlTypes.ts

```diff
@@ -0,0 +1,359 @@
+import { GraphQLResolveInfo } from 'graphql';
```

Comment:
We either need a way we can generate this on the command line 

OR

we need to check in the generated file

Otherwise we have no way to get a reliable version of it from a container. 😤

Reply:
Ohhhh @theyokohamalife and I figured it's better to point directly to the `schema.graphql` file. Using a live server is better for when you need to generate types for frontend code 💡